actors/eam/src/lib.rs

```diff
@@ -78,13 +74,15 @@ pub struct InitAccountParams {
 }
 
 #[derive(Serialize_tuple, Deserialize_tuple, Debug, PartialEq, Eq)]
-pub struct EamReturn {
+pub struct Return {
```

Comment:
(really nitty)

This may be a bit confusing if/when we add more return types. Personally, I'd just go with:

```rust
pub struct CreateReturn { ... }
pub type Create2Return = CreateReturn;
```